README.md

```diff
@@ -1 +1,38 @@
-# school-library
\ No newline at end of file
+# school-library
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> Create your own Enumerable.
+
+## Project requirements
+
+The project requirements can be viewed [here](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/create_your_own_enumerable.md)
+
+## Built With
+
+- Major languages: Ruby
+- Frameworks: N/A
+- Technologies used: N/A
+- Tested with: - [Rubocop](https://rubocop.org/)
+
+## Authors
+
+👤 **Victoir Barh**
+
+- GitHub: [@vic778](https://github.com/vic778)
+- Twitter: [@VictoirBarh](https://twitter.com/VictoirBarh)
+- LinkedIn: [Victoir barherenduba](https://www.linkedin.com/in/victor-emmanuel-barh-a93900200/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Hombre2014/Enumerable).
+
```

Comment:
- [ ] Please your `readme file` is about `Ruby enumerable`, please kindly update it to reflect the current milestone(`OOP school library: add Person, Student and Teacher classes`). Please make sure you check all the sections and update them appropriately as this will make your `readme file` more informative and professional.